### PR TITLE
DI-323 Fix standard and specified opening times data discrepancies

### PR DESCRIPTION
**Pull request by mik3ola**
## Link to JIRA Ticket
https://nhsd-jira.digital.nhs.uk/browse/DI-323
-

## Description

Specified opeining-times data validations - Includes testing different variations of the Specified opening-times data set.

### Noteworthy Changes

Created new function to create new variation of CE for specific tests

## Type of change

Delete not appropriate

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing

Please tick the testing that has been completed

- [ ] Integration tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Integration tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

File: test/integration/steps/test_parent_steps.py [hunk 1]
```diff
@@ -5,8 +5,13 @@
...
 
-from .utilities.events import build_same_as_dos_change_event, change_request, create_change_event, nhsuk_changed_event
-
+from .utilities.events import (
+    build_same_as_dos_change_event,
+    change_request,
+    create_change_event,
+    set_opening_times_change_event,
+)
+# from .utilities.encryption import initialise_encryption_client
```
Comment (JackPlowman) [inline test/integration/steps/test_parent_steps.py:14]: Can this line be deleted

**Review by JackPlowman — CHANGES_REQUESTED** [review]
One tiny change to make otherwise looks great

**Review by rohith1122 — APPROVED** [review]
LGTM

File: test/integration/steps/test_parent_steps.py (hunk 1 above)
Comment (mik3ola) [reply test/integration/steps/test_parent_steps.py]: Removed.

**Review by rohith1122 — APPROVED** [review]

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Pull request merged**